docs/Project.toml

```diff
@@ -0,0 +1,19 @@
+[deps]
+AbstractTrees = "1520ce14-60c1-5f80-bbc7-55ef81b5835c"
+DotEnv = "4dc1fcf4-5e3b-5448-94ab-0c38ec0385c1"
+EzXML = "8f5d6c58-4d21-5cfd-889c-e3ad7ee6a615"
+Gumbo = "708ec375-b3d6-5a57-a7ce-8257bf98657a"
+HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+Inflate = "d25df0c9-e2be-5dd7-82c8-3ad0b3e990b9"
+PromptingTools = "670122d1-24a8-4d70-bfce-740807c42192"
+Tar = "a4e569a6-e804-4fa4-b0f3-eef7a1d5b13e"
+URIParser = "30578b45-9adc-5946-b283-645ec420af67"
+URIs = "5c2747f8-b7ea-4ff2-ba2e-563bfd36b1d4"
+
+[compat]
+AbstractTrees = "0.4.5"
```

Comment:
Usually, you don't need to set compat in the docs of the project, because by depending on the main project, it will inherit those restrictions --> then you don't have to change it twice every time